web/templates/popovers/left_sidebar/left_sidebar_starred_messages_popover.hbs

```diff
@@ -14,7 +14,7 @@
                 <i class="popover-menu-icon zulip-icon zulip-icon-hide" aria-hidden="true"></i>
                 <span class="popover-menu-label">{{t "Hide starred message count" }}</span>
                 {{else}}
-                <i class="popover-menu-icon fa fa-eye" aria-hidden="true"></i>
+                <i class="popover-menu-icon zulip-icon zulip-icon-eye" aria-hidden="true"></i>
```

Comment:
Should we do a sweep for `fa-eye` usage.

Reply:
That would be great. No good reason to use multiple similar-looking eyes. :)